akka-remote/src/main/resources/reference.conf

```diff
@@ -14,12 +14,20 @@ akka {
 
     serializers {
       akka-containers = "akka.remote.serialization.MessageContainerSerializer"
+      akka-misc = "akka.remote.serialization.MiscMessageSerializer"
       proto = "akka.remote.serialization.ProtobufSerializer"
       daemon-create = "akka.remote.serialization.DaemonMsgCreateSerializer"
     }
 
     serialization-bindings {
       "akka.actor.ActorSelectionMessage" = akka-containers
+      # The classes akka.actor.Identify and akka.actor.ActorIdentity serialization/deserialization are required by
+      # the cluster client to work.
+      # For the purpose of preserving protocol backward compatibility, akka.actor.Identify and akka.actor.ActorIdentity
+      # are stil using java serialization by default.
+      # Should java serialization is disabled, uncomment the following lines
+      # "akka.actor.Identify" = akka-misc
+      # "akka.actor.ActorIdentity" = akka-misc
```

Comment:
@patriknw: serialization disabled by default for protocol backward compatibility - thanks for pointing this out!
@ktoso: issue raised to enable when backward compat is broken: #20395